### DISCUSSION TITLE
Docker container for onnxruntime-gpu

### DISCUSSION THREAD

**Discussion by shashwat623** [microsoft/onnxruntime#8704] [Other Q&A] [answered]
I have an application running on docker container which uses onnxruntime-gpu for inference. I would like to use the onxruntime cuda base docker image for the required cuda, cudnn and onnxruntime-gpu dependencies. I would want to know which version of onnxruntime-gpu does the latest base image use. I ask this because I am using onnxruntime-gpu python package for inference and  only the latest release of onnxruntime-gpu(1.8.1) provides inference flexibility for both GPU and CPU, so this is the minimum version I would need. Referring to the ONNX Runtime Docker Image tags [HERE](https://github.com/microsoft/onnxruntime/tree/master/dockerfiles), it shows that the latest image uses CUDA10.2, which makes me feel that the onnxruntime-gpu version used would be 1.5-1.6 as mentioned [HERE](https://onnxruntime.ai/docs/reference/execution-providers/CUDA-ExecutionProvider.html#requirements). Can someone please clarify this?

**Comment by harshithapv** [comment, marked as answer]
You would need to upgrade to CUDA 11.1 to use onnxruntime-gpu 1.8.1. The lastest-cuda image uses an CUDA 10.2 and that's why the version is 1.6. You can try building [this](https://github.com/microsoft/onnxruntime/blob/master/dockerfiles/Dockerfile.cuda) dockerfile for onnxruntime-gpu 1.8.1.

**Reply by snnn** [reply]
Sorry, it should be 11.0  for onnxruntime-gpu 1.8.x.

**Reply by snnn** [reply]
You can open https://github.com/microsoft/onnxruntime/blob/rel-1.8.1/tools/ci_build/github/azure-pipelines/templates/py-packaging-stage.yml and search "cuda_version"
